api/server.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const data = require('../data');
+
+const server = express();
+server.use(express.json());
+
+const GAMES_GET_ENDPOINT = '/games';
+const GAMES_POST_ENDPOINT = '/games';
+
+server.get('/', (req, res) => {
+    res.status(200).json({ api: 'active' });
+});
+
+server.get(GAMES_GET_ENDPOINT, (req, res) => {
+  const games = data.list();
+  res.status(200).json(games);
+});
+
+server.post(GAMES_POST_ENDPOINT, (req, res) => {
+    const game = req.body;
+    if (game.title && game.genre) {
```

Comment:
Then you could do 
```suggestion
    if (title && genre) {
```

Same thing with one less variable!